README.md

```diff
@@ -28,5 +119,5 @@ Input
 The signals should have attributes that can be evaluated by *key* and *value*.
 
 Output
-------
-For each input list of signals there is one output signal. It has an attribute for each *key* and that attribute is a list with a *value* for each corresponding input signal. When *group_attr* is set, it will be added as a value on the output signal with a value that is the group. If *one_value* is True, then the signal values are a single item instead of a list of all matching values.
+---------
```

Comment:
I think the "Output" section still makes sense, as a place to describe the output. "Examples" helps show the output. But I think the "Input" section above this could be removed in all the READMEs. What do you think?